examples/policies_definition.json

```diff
@@ -0,0 +1,25 @@
+{
+  "__help_sla" : "Add as many SLAs as you need. Format:",
+  "__help_sla" : "sla name, latency (ms), loss (%), jitter (ms)",
+  "sla_defs": [
+        ["video_calls_sla", "40", "2", "25"],
+        ["file_transfer_sla", "500","10", "100"]
+  ],
+
+  "__help_data" : "Add as many policies as you need. Format: service_directory_metadata_value, policy name, tunnel color, encapsulation (gre, ipsec)",
+  "data_policies": [
+          ["video", "prefer_mpls", "mpls","ipsec"],
```

Comment:
Since we have some public presentations already where we use public-internet and biz-internet, I would change the default here to be ["video", "prefer_biz_internet", "biz-internet","ipsec"], and then below prefer_public_internet. And then if we have other files in the scripts referencing these.